app/src/main/java/org/permanent/permanent/network/NetworkClient.kt

```diff
@@ -53,12 +53,13 @@ class NetworkClient {
         )
 
         val interceptor = HttpLoggingInterceptor()
-        interceptor.level = HttpLoggingInterceptor.Level.NONE
+        interceptor.level = if (BuildConfig.DEBUG) HttpLoggingInterceptor.Level.BODY
```

Comment:
this will cause large file uploads to fail in debug mode

Reply:
Oh, ok. I will remove it